### ISSUE TITLE
[BUG] google translate tabbing

### ISSUE THREAD

**Issue by kevindherman** [proudcity/wp-proudcity#2686]
## Source

* [Support ticket](https://proudcity.zendesk.com/agent/tickets/7580)

## Describe the bug
I noticed a bug with the translate dropdown menu that impacts accessibility. You cannot operate and select a different language from the dropdown using only the keyboard - https://www.w3.org/WAI/WCAG22/Understanding/keyboard.html.

1. Go to the website https://www.santa-ana.org/
2. Hit tab until you focus on the translate button
3. Hit enter to expand the dropdown
4. Use arrow keys to hover over another language
5. Hit Enter or Space. Neither select a new language
Link to page with this issue
https://www.santa-ana.org/
Upload screenshot of issue

* Translate_Dropdown_Accessibility.mp4 (https://storage.googleapis.com/proudcity/gravity_forms/4-e67a234e0ce0eb1fbe609819c16f2966/2025/09/Translate_Dropdown_Accessibility.mp4)

## QA

Notes:

* 

Links:

*

**Comment by curtismchale** [comment]
I submitted a support ticket with gtranslate the plugin author: https://wordpress.org/support/topic/language-menu-not-accessible-via-tab-or-arrow-keys/#new-topic-0

**Comment by curtismchale** [comment]
@kevindherman so far they say it works for them in Chrome on Windows. It does work for me on Linux via Chrome. It doesn't work in any browser on macOS and it doesn't work for me in Firefox on Linux.

I bet based on their initial response we either won't hear back or they'll say that their test in Chrome on Windows is sufficient and they're not going to do anything about it.

I don't see which browser or operating system the customer was using either.

**Comment by kevindherman** [comment]
Following up with the customer. 

I would think not working in macOS and/or Firefox would be enough for the plugin owner, clearly not.

**Comment by kevindherman** [comment]
@curtismchale Here's the customer info: 
Browser: Google chrome version 140.0.7339.186
OS: Windows 11 version 24H2

Maybe that'll get plugin owner's attention @curtismchale ?

**Comment by curtismchale** [comment]
@kevindherman they're still engaging, but also suggest that the paid version doesn't rely on the JS code that they think might not be working so we should look at upgrading to paid.

**Comment by curtismchale** [comment]
Tested a new update to the JS from the forums today and it still doesn't work.

**Comment by curtismchale** [comment]
@kevindherman the end result is that they say they tested on Windows and aren't going to put time into testing browsers on macOS or Linux so if it doesn't work we should be using the paid version.

**Comment by curtismchale** [comment]
Alternatives to gtranslate that we could evaluate.

- [Polylang](https://wordpress.org/plugins/polylang/) also has a [Pro](https://polylang.pro/) version
- [Weglot](https://wordpress.org/plugins/weglot/) 
- [Translate This](https://wordpress.org/plugins/conveythis-translate/)
- [WPML](https://wpml.org/) paid only

May be more but that's what I find right now.

**Comment by kevindherman** [comment]
@curtismchale this is something that may need to be prioritized this week. I think doing our own plugin at this point is likely the answer.

Just getting on your radar as you look ahead to the week ahead. 

Lets discuss tomorrow.

**Comment by curtismchale** [comment]
Ultimately this was fixed with releasing the https://github.com/proudcity/wp-proud-translations plugin. The main gtranslate plugin won't be fixing any tabbing.
